draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -570,24 +639,29 @@
 
         </section>
 
+	<!-- XXX JMH - the point below about "from router-X" ignores route
+	reflectors. Is this only intended to work when received directly? -->
```

Comment:
should be received either from Router-X or from RR with BGP Originator-ID being Router-X. 

Is the following text acceptable? 

_"When an ingress router receives a BGP UPDATE message containing the Metadata Path Attribute with the Site Physical Availability Index Sub-TLV from Router-X or its RR with the Originator-ID set to Router-X, the ingress router SHOULD set the next-hop to Router-X’s loopback address."_

Reply:
I think what you're saying is that yes, route reflectors can be used.
But the text you're proposing is that the ingress is actively changing BGP nexthops.  That's not normally how you want these features to work.  Normally, the originating router sets nexthop-self, the reflector doesn't change it.

Reply:
using "equal to" or "matching" instead of "set", Change the sentence to the following: 

"When an ingress router receives a BGP UPDATE message containing the Metadata Path Attribute with the Site Physical Availability Index Sub-TLV from Router-X or its RR with the Originator-ID equal to Router-X, the ingress router SHOULD set the next-hop to Router-X’s loopback address"

Reply:
Text is updated. Will likely need to review in the wg.